test/e2e/framework.go

```diff
@@ -29,9 +29,10 @@ import (
 )
 
 const (
-	defaultTimeout         = 100 * time.Second
+	defaultTimeout         = 300 * time.Second
```

Comment:
Does it mean static route need more time to realize?

Reply:
100s is not enough for some case. I met the case while did e2e test.